packages/devtools/frigg-cli/backendPath.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs-extra');
+const path = require('path');
+const PACKAGE_JSON = 'package.json';
+
+function findNearestBackendPackageJson() {
+    let currentDir = process.cwd();
+    while (currentDir !== path.parse(currentDir).root) {
+        const packageJsonPath = path.join(currentDir, 'backend', PACKAGE_JSON);
+        if (fs.existsSync(packageJsonPath)) {
+            return packageJsonPath;
+        }
+        currentDir = path.dirname(currentDir);
+    }
+    return null;
```

Comment:
should we return null? I would expect to see an error (eg throwed) if no package.json is found.

Reply:
I believe running npm install on the backend path will initialize and generate? Worth testing at some point, but also expecting that this will almost never run into this logic unless someone gets creative with their frigg app setup